instaclustr/Common_vpc_peering.go

```diff
@@ -0,0 +1,71 @@
+package instaclustr
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceVpcPeeringDelete(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*Config).Client
+
+	cluster, err := client.ReadCluster(d.Get("cluster_id").(string))
+	if err != nil {
+		return fmt.Errorf("[Error] Error retrieving cluster info: %s", err)
+	}
+	cdcID := cluster.DataCentres[0].ID
+	vpcPeeringID := d.Get("vpc_peering_id").(string)
+	log.Printf("[INFO] Deleting VPC peering connection %s.", vpcPeeringID)
+	err = client.DeleteVpcPeering(cdcID, vpcPeeringID)
+	if err != nil {
+		return fmt.Errorf("[Error] Error deleting VPC peering connection: %s", err)
+	}
+
+	d.SetId("")
+	d.Set("vpc_peering_id", "")
+	d.Set("cdc_id", "")
```

Comment:
**[Question]** What about setting the connection ID, or project ID as GCP term it?

Reply:
`peer_project_id` has been set in the `resource_gvp_vpc_peering.go` file as this was only GCP specific and not common